src/handlers/metadataDelta/handlers/addVerifiedMembers.ts

```diff
@@ -43,6 +46,17 @@ export const handleAddVerifiedMembers = async (
         id: getColonyContributorId(colonyAddress, userAddress),
       });
 
+      // If a wallet address is not currently a member of a Colony,
+      // add the entity as a new Colony contributor
+      if (item?.data?.getColonyContributor === null) {
+        await createColonyContributor({
```

Comment:
I don't think you can simply reuse this function.

It creates a contributor, but it sets `isVerified` to false and, potentially worse, `hasPermissions` to true.